<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindowClass</class>
 <widget class="QMainWindow" name="CMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1545</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>闻泰中框检测软件_V1.0,0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="CMainWindow.qrc">
    <normaloff>:/CMainWindow/Resources/app.ico</normaloff>:/CMainWindow/Resources/app.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_Cam1">
      <property name="title">
       <string>工位一</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_Image1">
         <property name="styleSheet">
          <string notr="true">background-image: url(:/CMainWindow/Resources/background.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_Cam2">
      <property name="title">
       <string>工位二</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_Image2">
         <property name="styleSheet">
          <string notr="true">background-image: url(:/CMainWindow/Resources/background.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_Cam3">
      <property name="title">
       <string>工位三</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_Image3">
         <property name="styleSheet">
          <string notr="true">background-image: url(:/CMainWindow/Resources/background.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QGroupBox" name="groupBox_Cam4">
      <property name="title">
       <string>工位四</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_Image4">
         <property name="styleSheet">
          <string notr="true">background-image: url(:/CMainWindow/Resources/background.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>明细</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_1">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>工位一</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>工位二</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>工位三</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>工位四</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Start"/>
   <addaction name="action_Stop"/>
   <addaction name="action_Setting"/>
  </widget>
  <action name="action_Start">
   <property name="icon">
    <iconset resource="CMainWindow.qrc">
     <normaloff>:/CMainWindow/Resources/start.png</normaloff>:/CMainWindow/Resources/start.png</iconset>
   </property>
   <property name="text">
    <string>开始检测</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="icon">
    <iconset resource="CMainWindow.qrc">
     <normaloff>:/CMainWindow/Resources/stop.png</normaloff>:/CMainWindow/Resources/stop.png</iconset>
   </property>
   <property name="text">
    <string>停止检测</string>
   </property>
  </action>
  <action name="action_Setting">
   <property name="icon">
    <iconset resource="CMainWindow.qrc">
     <normaloff>:/CMainWindow/Resources/settings.png</normaloff>:/CMainWindow/Resources/settings.png</iconset>
   </property>
   <property name="text">
    <string>参数设置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CMainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
